tests/proquest/test_client.py

```diff
@@ -310,8 +310,8 @@ def test_get_book_correctly_fails(
     def test_get_book_correctly_extracts_open_access_books(self):
         # Arrange
         book_content = "PDF Book12345"
-        response_arguments = {"content": book_content}
-        expected_open_access_book = ProQuestBook(content=bytes(book_content))
+        response_arguments = {"content": book_content.encode("utf-8")}
+        expected_open_access_book = ProQuestBook(content=book_content.encode("utf-8"))
```

Comment:
Not a big deal but it would make more sense conceptually to have the mock book contents start out as bytestrings, since they represent files on disk.